ipyaggrid/js/helpersBuiltin.js

```diff
@@ -84,22 +84,6 @@ helpersBuiltin = {
      * @param {Object} gridOptions
      */
     exportToExcel: function(gridOptions) {
-        const params = {
-            skipHeader: false,
-            columnGroups: true,
-            skipFooters: false,
-            skipGroups: false,
-            skipPinnedTop: false,
-            skipPinnedBottom: false,
-            allColumns: true,
-            onlySelected: false,
-            suppressQuotes: true,
-            fileName: 'my_file.xls',
-            sheetName: 'my_sheet',
-            shouldRowBeSkipped: () => false, // TODO : check
-            processCellCallback: param => param.value,
-            processHeaderCallback: null,
-        };
-        gridOptions.api.exportDataAsExcel(params);
+        gridOptions.api.exportDataAsExcel();
```

Comment:
What `params` are effectively changed by removing the default `params` struct?

Reply:
Those that I removed are not the default `params` struct.  Those parameters look like work-in-progress that were not followed through.  This PR re-establishes the real defaults.

Reply:
Although some params don't seem to be supported anymore and others just are the default anyway, the params allColumns, fileName and sheetName are used. We don't want to break existing users who may rely on these settings now.

I think we could add an attribute `export_excel_options` to `Grid`, and use those if set to overwrite the current defaults.

Fixing https://github.com/widgetti/ipyaggrid/issues/47 can be done in a separate PR only adding `x` to the fileName, keeping the other params the same.

Reply:
Users who rely on the existing values can still explicitly set them.

This PR is really about correcting the initial error of trying to specify a set of default parameters that is different from the one specified by the JS library, leading to problems (unsupported parameters, wrong extensions, change for the sake of change) without any really obvious advantage.

Reply:
Yeah, but it's still be a breaking change and an end-user might not be able to change the code. However I overlooked ipyaggrid is still at 0.x versions, so it might be acceptable to make breaking changes.

Reply:
I don't see how values could be explicitly set though.

Reply:
The way to do it is, for instance:

```
Grid(
    ...,
    grid_options={
        ...,
        "js_helpers_custom":
             "helpers.exportToExcel = opts => opts.api.exportDataAsExcel({sheetName: 'my_sheet'});",
    }
)
```

Reply:
Ah, right, you can just overwrite the builtin helpers.